lib/main_common.pm

```diff
@@ -1400,7 +1400,6 @@ sub load_extra_tests_y2uitest_ncurses {
     unless (is_s390x) {
         loadtest "console/yast2_proxy";
         loadtest "console/yast2_vnc";
-        loadtest "console/yast2_samba";
```

Comment:
this could be excluded based on a variable. The reason is that is will be manageable letting others knows whats going on and it will be easier to recover. What do you think?

Reply:
we could just mention in the follow-up ticket not forget to re-enable in O3 with proper schedule.